api/controllers/UserController.js

```diff
@@ -105,7 +105,7 @@ module.exports = {
   },
 
   status: async function (req, res) {
-    const { usr: id, loggedIn: authenticated } = req.session;
+    const { usr: id, loggedIn: authenticated, game } = req.session;
```

Comment:
This might be clearer if it were called `gameId`. I know that's coming directly from the session but it's a little funky seeing it here without typescript telling us whether that's a number or obj

Reply:
I originally had it that way, I kind of wish the schema called it that too.